Rakefile

```diff
@@ -354,10 +309,104 @@ Failed to create the codesigning keychain:
   desc "Roll back changes to DeviceAgent binaries; does not touch client.rb"
   task :checkout do
     banner("Git Checkout")
-    checkout(bin)
-    checkout(frameworks_zip)
     checkout(app_zip)
     checkout(ipa_zip)
   end
 end
 
+namespace :iOSDeviceManager do
```

Comment:
```suggestion
namespace :idm do
```

Reply:
Thanks.
`iOSDeviceManager` namespace was renamed to `idm`